### PR TITLE
Guiding principles

### PR DESCRIPTION
**Pull request by corvus-ch**

### CODE REVIEW

**Comment by tobru** [comment]
Fixed some typos and reworded a few sentences. The content LGTM so far.

File: docs/modules/ROOT/pages/references/principles.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  In the case Kyverno is not suitable, we will create a custom admission/mutating admission webhook for exactly that use case.
+
+* In cases where data needs to be copied to other locations, we will only do so in one direction.
+  
+* Data owned by Keycloak will be cached on each Zone.
+  This for once is required for Kyverno to work.
+  It also makes the cluster more resilient against connectivity issues between Keycloak and the Zone.
+
+* We will enforce quota and limits on Zone level.
+  As it is our main goal to protect the zone (Cluster) from abuse.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/references/principles.adoc:18]: This feels like a random security requirement out of the blue. I would propose to move this into the section where we document quality requirements (format and location tbd, let me think of something...)

File: docs/modules/ROOT/pages/references/principles.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+** Users
+
+* We make use of https://kyverno.io/[Kyverno] to enforce policies that go beyond the capabilities of RBAC.
+  In the case Kyverno is not suitable, we will create a custom admission/mutating admission webhook for exactly that use case.
+
+* In cases where data needs to be copied to other locations, we will only do so in one direction.
+  
+* Data owned by Keycloak will be cached on each Zone.
+  This for once is required for Kyverno to work.
+  It also makes the cluster more resilient against connectivity issues between Keycloak and the Zone.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/references/principles.adoc:15]: I would argue that this isn't a principle, but rather in general our current system idea/proposal. I'd like to come up with a different proposal and a diagram supporting this text.

**Comment by tobru** [comment]
Replaced by #5

**Pull request closed without merging**